### PR TITLE
Fix inconsistencies in RFC

### PR DESCRIPTION
**Pull request by dplumb94**
Fixes several small issues with the RFC

### CODE REVIEW

File: text/0025-purchase-order.md [hunk 1]
```diff
@@ -551,8 +552,9 @@ message PurchaseOrderPayload {
 }
 
 message CreatePurchaseOrderPayload {
```
Comment (shannynalayna) [inline text/0025-purchase-order.md:555]: Doesn't this message also need an optional `CreateVersionPayload`?

File: text/0025-purchase-order.md [hunk 2]
```diff
@@ -551,8 +552,9 @@ message PurchaseOrderPayload {
...
-  string uuid = 1;
-  uint64 created_at = 2;
+  string org_id = 1;
+  string uuid = 2;
+  uint64 created_at = 3;
 }
 
 message UpdatePurchaseOrderPayload {
```
Comment (shannynalayna) [inline text/0025-purchase-order.md:561]: Would this message also need a `po_uuid`?

Comment (dplumb94) [reply text/0025-purchase-order.md:561]: That is just the `uuid` here

File: text/0025-purchase-order.md (hunk 1 above)
Comment (dplumb94) [reply text/0025-purchase-order.md:555]: Ah yeah I was not originally going to make that change here, but I'll add it.

File: text/0025-purchase-order.md (hunk 2 above)
Comment (shannynalayna) [reply text/0025-purchase-order.md:561]: I'm slightly confused where the `uuid` is coming from in this message?

**Review by shannynalayna — APPROVED** [review]

File: text/0025-purchase-order.md [hunk 3]
```diff
@@ -638,31 +638,30 @@ Validation Requirements
...
-Address of Grid Purchase Order `621dee05`
+Address of Grid Purchase Order `621dee06`
 
 ### Update Version Payload
 
 `UpdateVersionPayload` updates an existing `PurchaseOrderVersion`
 with a new revision.
 
 Validation Requirements:
-- The `org_id` must exist in Pike for it be a valid transaction
```
Comment (vaporos) [inline text/0025-purchase-order.md:654]: This looks unrelated to this commit.

Comment (dplumb94) [reply text/0025-purchase-order.md:654]: moved to the proper commit

**Review by shannynalayna — APPROVED** [review]

**Review by vaporos — APPROVED** [review]

**Review by shannynalayna — APPROVED** [review]

File: text/0025-purchase-order.md [hunk 4]
```diff
@@ -566,8 +566,9 @@ message UpdatePurchaseOrderPayload {
 
 message CreateVersionPayload {
```
Comment (shannynalayna) [inline text/0025-purchase-order.md:568]: In the same vein as this commit, I believe the `UpdatePurchaseOrderPayload` needs a `po_uid` field

File: text/0025-purchase-order.md [hunk 5]
```diff
@@ -566,8 +566,9 @@ message UpdatePurchaseOrderPayload {
...
   string version_id = 1;
-  bool is_draft = 2;
-  PayloadRevision revision = 3;
+  string po_uid = 2;
+  bool is_draft = 3;
+  PayloadRevision revision = 4;
 }
 
 message UpdateVersionPayload {
```
Comment (shannynalayna) [inline text/0025-purchase-order.md:575]: Also, this payload needs a `po_uid` to access the purchase order in state
